### DISCUSSION TITLE
Some course website bug

### DISCUSSION THREAD

**Discussion by majauhar** [OpenMined/courses#247] [Feedback] [answered]
Browser: Chrome Version 88.0.4324.146 (Official Build) (x86_64)
System: macOS Big Sur 11.1 (20C69)

I am guessing this issue has already been reported. Just wanted to drop it again just in case.
![Screenshot 2021-02-09 at 10 23 29 PM](https://user-images.githubusercontent.com/43479350/107398877-14fd3580-6b26-11eb-9d7d-ddcfb5bf067f.png)

**Comment by hwrdtm** [comment, marked as answer]
Thanks @majauhar for reporting this. This is already tracked in https://github.com/OpenMined/openmined/issues/96.

**Reply by majauhar** [reply]
You are welcome @hwrdtm . I didn't look into the open issues earlier. I see it's being fixed. The course website is otherwise great.  Kudos to the team 🙌🏼
